components/script/dom/gpuconvert.rs

```diff
@@ -6,8 +6,10 @@ use std::borrow::Cow;
 use std::num::NonZeroU64;
 
 use webgpu::wgc::command as wgpu_com;
+use webgpu::wgc::pipeline::ProgrammableStageDescriptor;
```

Comment:
I wanted to look up the documentation for this struct, but I had to grep through the source code, because I forgot what `wgc` was an alias for. This is a little frustrating.  Maybe we can reconsider these aliases?

Reply:
Opened https://github.com/servo/servo/issues/33300 for discussion.

Reply:
Thanks. I appreciate it!